docs/AUTH.md

```diff
@@ -0,0 +1,25 @@
+# Authentication
+
+Authentication can be used as a guard on a field, query or mutation, restricting data access or actions for a specific group of users.
+
+Since the codebase uses TypeGraphQL, which relies heavily on decorators, authentication is also done using decorators.
+
+The auth logic is mostly ported from v3 API ([hpc_service](https://github.com/UN-OCHA/hpc_service)) and available for use with two decorators:
+
+## `Authorized()`
+
+This decorator is [natively supported](https://typegraphql.com/docs/authorization.html) by TypeGraphQL and enables to restrict access to logged-in users only.
+
+When used with parameters, this decorator can be used to require users to have one of the listed roles. For example, decorating with `@Authorized('projectsAdmin')` allows only projects admin to access resource.
+
+**Note: Only global roles can be used with this decorator.**
```

Comment:
I'm not sure that we have a concrete need for this decorator. If something requires a authorized acces, then it should require a specific permission, if not then it should be open to anyone to access (subject to rate-limiting of course, but that's a different matter).

I also see a potential risk that developers may incorrectly use `Authorized` instead of `Permission` in a number of cases (ie. new developers that have not yet fully familiarized themselves with the system).

Reply:
Removed `@Authorized()` decorator altogether.